src/libclipper/include/clipper/containers.hpp

```diff
@@ -5,14 +5,15 @@
 #include <memory>
 #include <unordered_map>
 
+#include <boost/circular_buffer.hpp>
 #include <boost/thread.hpp>
 
 #include <clipper/datatypes.hpp>
 #include <clipper/util.hpp>
 
 namespace clipper {
 
-using Deadline = std::chrono::time_point<std::chrono::high_resolution_clock>;
+using Deadline = std::chrono::time_point<std::chrono::system_clock>;
 
```

Comment:
Based on some stackoverflow research, it seems that `high_resolution_clock` is inconsistent across platforms. There is no consistent epoch for `high_resolution_clock`, and most systems don't actually support the nanosecond temporal resolution that this type of clock was created to expose. Many systems use `high_resolution_clock` as an alias for `system_clock`, but not all behave this way (i.e. OSX). Because `system_clock` is consistent, I think we should use it throughout Clipper. If you agree, I can address that change in a separate PR.